lambdanotes/src/components/App.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import './App.css';
+import CreateNote from './CreateNote';
+import NoteList from './NoteList';
+import { Link } from 'react-router-dom';
+
+class App extends Component {  
```

Comment:
Always reflect your component names as to what their function is. I see an App component here that has a sidebar in it...and that's it. So I would call this Sidebar component instead of App component. I would expect an App component to encompass all the main components of the App or to direct traffic via React Router. Clean code otherwise!